packages/telemetry/package.json

```diff
@@ -13,7 +13,10 @@
   "bugs": "https://github.com/withastro/astro/issues",
   "homepage": "https://astro.build",
   "exports": {
-    ".": "./dist/index.js",
+    ".": {
+      "default": "./dist/index.js",
+      "types": "./dist/types/index.d.ts"
```

Comment:
The `"types"` need to come before `"default"` since exports condition is order-sensitive.